src/mission.h

```diff
@@ -198,7 +209,7 @@ struct mission_type {
     std::function<void( mission * )> end = mission_end::standard;
     std::function<void( mission * )> fail = mission_fail::standard;
 
-    std::map<std::string, std::string> dialogue;
+    std::map<std::string, std::string> mdialogue;
```

Comment:
Not sure why you changed this. Is it because it would shadow the global type `dialogue` and make it difficult to refer to that type from within the scope of `mission_type`? Note that you can always use the scope operator to clear this:
```C++
class dialogue {
};
class other {
    std::string dialogue();
    void foo() {
         ::dialogue instance_of_that_type;
         dialogue(); // call to that function
    }
};
```

Reply:
Yes, exactly.